package.json

```diff
@@ -21,8 +21,8 @@
   "author": "Leouarz",
   "devDependencies": {
     "@polkadot/api": "^12",
-    "@subql/cli": "^5.2.4",
-    "@subql/node": "^5.2.3",
+    "@subql/cli": "5.2.8",
+    "@subql/node": "5.2.7",
     "@subql/types": "3.11.2",
```

Comment:
Should we bump types as well?
New version was just released 2hrs ago.

Reply:
Yep, tried it and it's still not usable and codegen fails due to mixed deps, and since we don't need it, let's just ship this and update when required in few weeks / month hopefully